internal/devbox/generate/tmpl/DevboxImageDockerfile

```diff
@@ -21,8 +21,21 @@ RUN . ~/.nix-profile/etc/profile.d/nix.sh
 ENV PATH="/home/${DEVBOX_USER}/.nix-profile/bin:$PATH"
 
 # Step 3: Installing devbox
-ENV DEVBOX_USE_VERSION=$DEVBOX_USE_VERSION
+ENV DEVBOX_USE_VERSION ${DEVBOX_VERSION}
 RUN wget --quiet --output-document=/dev/stdout https://get.jetify.com/devbox   | bash -s -- -f
 RUN chown -R "${DEVBOX_USER}:${DEVBOX_USER}" /usr/local/bin/devbox
+# stop launcher from auto updating
+RUN mkdir -p $HOME/.config/devbox/ && echo "false" > $HOME/.config/devbox/auto_update
 
-CMD ["devbox", "version"]
+# run a devbox command to make launcher download devbox binary
+RUN devbox version
+
+# replace binary with launcher
+USER root
+RUN if [ $(uname -m) = "aarch64" ]; \
+    then mv -f /home/devbox/.cache/devbox/bin/${DEVBOX_VERSION}_linux_arm64/devbox /usr/local/bin; \
+    else mv -f /home/devbox/.cache/devbox/bin/${DEVBOX_VERSION}_linux_amd64/devbox /usr/local/bin; \
+    fi
```

Comment:
We publish images for both `arm64` and `amd64` in the same github action. The devbox binary location changes based on cpu architecture. So this conditional `RUN` command is necessary.